app/helpers/lists_helper.rb

```diff
@@ -9,10 +9,26 @@ def list_item list, active = false
       "aria-controls" => list.id 
     }
     link_to("#list-#{list.id}", options) do
-      list_name = content_tag :div, list.name, class: "me-auto list-name"
+      if is_shared_list?(list, current_user)
+        list_name = content_tag :div, list.name + " (shared)", class: "me-auto list-name"
+      else
+        list_name = content_tag :div, list.name, class: "me-auto list-name"
+      end
```

Comment:
Since the only thing changing here is the display value, lets decrease the amount of duplication and set a variable for only the content and then use that variable in the content tag.
```suggestion
      display_name = list.name
      if is_shared_list?(list, current_user)
        display_name << " (shared)"
      end
      list_name = content_tag :div, display_name, class: "me-auto list-name"
```